core/main/src/Utils/CanvasUtils.ts

```diff
@@ -460,4 +460,33 @@ export class CanvasUtils {
             context.restore();
         }
     }
+
+    public static drawEllipse(
+        context: CanvasRenderingContext2D,
+        particle: IParticle,
+        fillColorValue: IHsl | undefined,
+        radius: number,
+        opacity: number,
+        width: number,
+        rotation: number
+    ): void {
+        const pos = particle.getPosition();
+        context.beginPath();
+
+        if (fillColorValue) {
+            context.strokeStyle = ColorUtils.getStyleFromHsl(fillColorValue, opacity);
+        }
+
+        if (width === 0) {
+            return;
+        }
+
+        context.lineWidth = width;
+
+        const rotationRadian = (rotation + 90) * (Math.PI / 180);
```

Comment:
Why `rotation + 90` instead of `rotation`?

Reply:
The default behaviour at 0 is a vertical ellipse instead of horizontal, like you asked.

Reply:
Oh ok, My bad I was misunderstood. The `rotation` value should be represented as is set. If `0` is set, a horizontal line should be drawn. In the preset that you created for testing the feature the value was `0` but the orbit was drawn with a 45° angle.

Reply:
The current behaviour is that `rotation` has a default value of `45` and will show as `45` in the editor. 
If you change that value to 0, a horizontal ellipse will be drawn. 
If I remove the `+90`, default behaviour will still be 45. But at 0, a vertical ellipse will be drawn instead. And at 90, a horizontal one will be drawn. Should I remove?

Reply:
Yes remove the 90, I've seen 0 and a 45-degree orbit in the test I did some commits ago, probably it's fine now. Once fixed, I'll test it again.